apps/src/templates/sectionsRefresh/CurriculumQuickAssign.jsx

```diff
@@ -88,25 +113,17 @@ export default function CurriculumQuickAssign() {
       {marketingAudience && courseOfferings && (
         <div>
           <h5>{marketingAudience}</h5>
-          {JSON.stringify(courseOfferings[marketingAudience])}
+          <QuickAssignTable
+            marketingAudience={marketingAudience}
+            courseOfferings={courseOfferings}
```

Comment:
This might be something you're already planning -- in which case, don't worry about it -- but do you need to pass in the whole blob here or could you just pass in `courseOfferings[marketingAudience]`?

Reply:
I am planning to do all the rendering in the table component, so will probably pass along the offerings too. Happy to discuss though... there might be an interesting conversation here around where we want the load to happen